apps/web/src/components/Shared/GlobalModals.tsx

```diff
@@ -28,9 +29,9 @@ const GlobalModals: FC = () => {
     setShowStatusModal,
     showProfileSwitchModal,
     setShowProfileSwitchModal,
-    showNewPostModal,
-    setShowNewPostModal,
+    showNewPublicationModal,
     showAuthModal,
+    setShowNewPublicationModal,
```

Comment:
put in order

showNewPublicationModal,
setShowNewPublicationModal

Reply:
Updated